lib/webpacker/dev_server_runner.rb

```diff
@@ -30,6 +30,7 @@ def load_config
         @port              = dev_server.port
         @pretty            = dev_server.pretty?
         @https             = dev_server.https?
+        @hot               = dev_server.hmr?
```

Comment:
Recent docs on using HMR with the webpack-dev-server all suggest using the command line option rather than the plugin.